api/tests/test_collection.py

```
@@ -0,0 +1,1845 @@
+from fastapi.testclient import TestClient
+from sqlalchemy.orm import Session
+
+from app.main import app
+from app.db import get_db, SessionLocal
+from app.db_models import Game, CollectionItem, Playthrough
+from app.schemas import AcquisitionType
+
+client = TestClient(app)
+
+# Ensure tables exist
+from app.db import Base
```

Ruff (0.12.2)

Module level import not at top of file

(E402)